internal/usage/usage.go

```diff
@@ -433,4 +433,5 @@ dbName and collection are required only for built-in roles.`
 	UpdateWarning                             = " Passing this flag replaces preexisting data."
 	EnableWatch                               = "Watch the command until it completes its execution or the watch times out. To set the time that the watch times out, use the --watchTimeout option."
 	WatchTimeout                              = "Time in seconds until a watch times out. After a watch times out, the CLI no longer watches the command."
+	CompactResponse                           = "Flag that enables the compact array response structure for json output on applicable list commands."
```

Comment:
A couple things... I think we can indicate here what Wojciech was saying (that this can enable legacy compatibility, although it can be used either way). Also, we should give more details on what compact array means and say they have to use --output json to use this. Please let me know if I've misunderstood exactly what the compact option means. Finally, I think we can take out "on applicable list commands" as it will only show for commands where it is applicable.
```suggestion
	CompactResponse                           = "Flag that enables the compact array response structure for a json output. The --compact option returns array objects as top-level responses and allows backward compatibility for scripts based on previous CLI versions. Omitting the --compact option for a json output returns array objects within a 'results' sub-array. You must specify --output json to use this option."
```

Reply:
Also let me know if this is too long, If so we could lose the first sentence I think and start with "Flag that enables the return of array objects as top-level..."

Reply:
I'm happy with this. Thanks for the suggestion!